pages/workflow/WorkflowHandler.php

```diff
@@ -208,142 +178,6 @@ protected function getEditorAssignmentNotificationTypeByStageId($stageId)
-                $decisionTypes = [
-                    new RequestRevisions(),
-                    new Accept(),
-                ];
-                $cancelReviewRound = new CancelReviewRound();
-                if ($cancelReviewRound->canRetract($submission, $reviewRoundId)) {
-                    $decisionTypes[] = $cancelReviewRound;
-                }
-                if ($submission->getData('status') === Submission::STATUS_DECLINED) {
-                    $decisionTypes[] = new RevertDecline();
-                } elseif ($submission->getData('status') === Submission::STATUS_QUEUED) {
-                    $decisionTypes[] = new Decline();
-                }
-                break;
-            case WORKFLOW_STAGE_ID_EDITING:
-                $decisionTypes = [
-                    new SendToProduction(),
-                    new BackFromCopyediting(),
-                ];
-                break;
-            case WORKFLOW_STAGE_ID_PRODUCTION:
-                $decisionTypes = [
-                    new BackFromProduction(),
-                ];
-                break;
-        }
-
-        Hook::call('Workflow::Decisions', [&$decisionTypes, $stageId]);
-
-        return $decisionTypes;
-    }
-
-    protected function getStageRecommendationTypes(int $stageId): array
-    {
-        switch ($stageId) {
-            case WORKFLOW_STAGE_ID_INTERNAL_REVIEW:
-                $decisionTypes = [
-                    new RecommendRevisionsInternal(),
-                    new RecommendAcceptInternal(),
-                    new RecommendDeclineInternal(),
-                    new RecommendSendExternalReview(),
-                ];
-                break;
-            case WORKFLOW_STAGE_ID_EXTERNAL_REVIEW:
-                $decisionTypes = [
-                    new RecommendRevisions(),
-                    new RecommendAccept(),
-                    new RecommendDecline(),
-                ];
-                break;
-            default:
-                $decisionTypes = [];
-        }
-
-
-        Hook::call('Workflow::Recommendations', [$decisionTypes, $stageId]);
-
-        return $decisionTypes;
-    }
-
-    protected function getPrimaryDecisionTypes(): array
-    {
-        return [
-            SkipInternalReview::class,
-            SendExternalReview::class,
-            AcceptFromInternal::class,
-            Accept::class,
-            SendToProduction::class,
-        ];
-    }
-
-    protected function getWarnableDecisionTypes(): array
-    {
-        return [
-            InitialDecline::class,
-            DeclineInternal::class,
-            Decline::class,
-            CancelInternalReviewRound::class,
-            CancelReviewRound::class,
-            BackFromCopyediting::class,
-            BackFromProduction::class,
-        ];
-    }
-
     protected function getTitleAbstractForm(string $latestPublicationApiUrl, array $locales, Publication $latestPublication, Context $context): TitleAbstractForm
```

Comment:
Same here.. the logic to get these forms is moved to SubmissionController and it provides the form config from the API.